docs/app-integration-development/20-OAuth-2-Extended-OAuth.md

```diff
@@ -0,0 +1,67 @@
+---
+tags: [app-integration-development]
+---
+
+# Extended OAuth
+
+<!-- theme: warning -->
+> ### Early Access
+> 
+> The features described on this page are in an Early Access state and are subject to change. Your PagerDuty Account may 
+> require a feature flag before this functionality is available to you. Please reach out to us if you have any questions or 
+> need support.
+
+## Register an App
+Extended OAuth Clients allow your application to act on a PagerDuty Account as a PagerDuty App. The access your application has to the PagerDuty Account is controlled by the scopes it is granted. Before you start building, you first need to register a PagerDuty App with an Extended OAuth Client. This is done via the Developer Mode UI in your PagerDuty Account.
```

Comment:
Should we call it as the "App Registration UI"  since we are renaming it anyway?